src/Dfc.ProviderPortal.TribalExporter/Functions/ProviderMigrator.cs

```diff
@@ -0,0 +1,356 @@
+﻿using CsvHelper;
+using Dfc.CourseDirectory.Models.Models.Providers;
+using Dfc.ProviderPortal.Packages.AzureFunctions.DependencyInjection;
+using Dfc.ProviderPortal.TribalExporter.Interfaces;
+using Dfc.ProviderPortal.TribalExporter.Models.Tribal;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using UkrlpService;
+
+namespace Dfc.ProviderPortal.TribalExporter.Functions
+{
+    public static class ProviderMigrator
+    {
+        [FunctionName(nameof(ProviderMigrator))]
+        [NoAutomaticTrigger]
+        public static async Task Run(
+                    string input,  // Work around https://github.com/Azure/azure-functions-vs-build-sdk/issues/168
+                    ILogger log,
+                    [Inject] IConfigurationRoot configuration,
+                    [Inject] IProviderCollectionService providerCollectionService,
+                    [Inject] ICosmosDbHelper cosmosDbHelper,
+                    [Inject] IBlobStorageHelper blobhelper,
+                    [Inject] IUkrlpApiService ukrlpApiService
+                    )
+        {
+            const string WHITE_LIST_FILE = "ProviderWhiteList-Jav.txt";
```

Comment:
Hey Jav, I think we should be using the same whitelist file across the board.

Reply:
Yes noticed that ! have changed it already :)